app/help/en/import.yml

```diff
@@ -3,4 +3,7 @@ import:
         Import data objects from specifications
 
     description: |
-        TODO
+        Import data objects from specifications
```

Comment:
Can just remove this line and similar duplicates of "overview" field

Reply:
Got it. Made those changes.